src/errorpage.cc

```diff
@@ -428,7 +441,7 @@ TemplateFile::loadFromFile(const char *path)
 
     if (fd < 0) {
         /* with dynamic locale negotiation we may see some failures before a success. */
-        if (!silent && templateCode < TCP_RESET) {
+        if (!silent && templateCode < TCP_RESET && !FindHardCodedTemplate(templateCode, TemplateDefaults)) {
```

Comment:
Let's remove this currently-unused change since we do not fully understand what/why TemplateDefaults is. Those who do will improve this code.

```suggestion
        if (!silent && templateCode < TCP_RESET) {
```

Reply:
Done.